src/main/java/uk/gov/hmcts/divorce/common/event/SubmitJointConditionalOrder.java

```diff
@@ -55,7 +55,8 @@ public void configure(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
             .forStates(ConditionalOrderDrafted, ConditionalOrderPending)
             .name("Submit Conditional Order")
             .description("Submit Conditional Order")
-            .endButtonLabel("Save Conditional Order")
+            .endButtonLabel("Submit Application")
+            .showSummary()
```

Comment:
We only seem to be able to set the button text when there's an event summary/confirmation page. I've flagged this up with Kim/Fubara in the ticket and can revert back to how this was before if they don't like the confirmation screen